### ISSUE TITLE
Analyze current support for multiple curation concerns (or work types)

### ISSUE THREAD

**Issue by mjgiarlo** [projecthydra/sufia#2411]
If I create a second curation concern via the work generator, what happens? Needs testing in the UI. Do things break? Are the 2nd through nth curation concerns ignored/hidden? Do they magically work (not likely)? We should document what the current behavior is so we can ticketize the remaining work.

Also needs revisiting once #2243 is merged.

Related to #2199

**Comment by mjgiarlo** [comment]
Ping @hackmastera

**Comment by hackmastera** [comment]
I generated a new work type called 'snowflake'.

- There are 2 'Create work' links on the dashboard. They share a hover animation but can be clicked on separately. They link to different routes (/concern/generic_works/new and /concern/snowflakes/new).
- There are also 2 'New Work' links under the 'Works' menu item in the top bar. 
- On each 'new work' page there is no indication of what kind of work you're creating. Expect 'Create new Work' and 'Create new Snowflake'.
- On Create new Snowflake page metadata help text is missing.
- On 'My Works' works listing, the filter by work type groups both objects into the same category, called 'Work'. Expect one for 'Work' and one for 'Snowflake'. Same on search results page.
- In Batch Create there is no way to tell what kind of work will be created. Should 'resource type' be tied to 'work type' in some way? Should 'work type' be added to the files upload form? After uploading I can see that both were created as Snowflakes.

I tested create work, batch create, and search. I added some works to a collection. I poked around a little. I didn't run into any fatal errors. Let me know if there is some other functionality you're concerned or curious about.

**Comment by jcoyne** [comment]
The text isn't changing because the existing translation for that button:
https://github.com/projecthydra/sufia/blob/master/config/locales/sufia.en.yml#L104

Doesn't make use of the work_type variable that is passed in:
https://github.com/projecthydra/sufia/blob/master/app/views/dashboard/_index_partials/_heading_actions.html.erb#L12

**Comment by mjgiarlo** [comment]
@hackmastera :tada: Would you mind creating a separate issue for each bullet? If you could also edit the description of #2199 to include the new issues in the tasklist there, that'd be swell. (I review all issues, so I will be happy to augment/label/milestone the issues you create.)

**Comment by hackmastera** [comment]
created and linked. close this one?

**Comment by mjgiarlo** [comment]
@hackmastera Got it. Thanks!
